.github/pull_request_template.md

```diff
@@ -1,36 +1,36 @@
 <!--
 Thank you for your contribution! 👍
-Please make sure to read CONTRIBUTING.md if you have not already.
-Disclaimer: Pull Requests that do not comply with the rules will be arbitrarily closed.
+Please make sure to read CONTRIBUTING.md if you have not already. Pull Requests that do not comply with the rules will be arbitrarily closed.
 -->
 
 ### 📝 Description
 
-_Replace this text by a clear and concise description of what this pull request is about and why it is needed._
+_Replace this text by a clear and concise description of what this pull request is about and why it is needed. Be sure to explain the problem you're addressing and the solution you're proposing._
+_For visual features, please attach screenshots or video recordings to demonstrate the changes._
+_For libraries, you can add a code sample of how to use it._
+_For bugfixes, you can explain the previous behavior and how it was fixed._
```

Comment:
Could be worthwhile to add some templates as (hidden) comments to easily / consistently display visual changes.

Examples of PRs descriptions where these have been used in the past:
- For images: https://github.com/LedgerHQ/ledger-live/pull/5069
- For videos: https://github.com/LedgerHQ/ledger-live/pull/4790 

```suggestion
_For bugfixes, you can explain the previous behavior and how it was fixed._

<!--
You can use the templates bellow to clearly display visual changes:

For videos

| Before  | After |
| ------------- | ------------- |
| <video src="https://user-images.githubusercontent.com/13440061/129455220-23fa27a2-c8f0-4953-b291-b4893959d5d9.mp4">  | <video src="https://user-images.githubusercontent.com/13440061/129455220-23fa27a2-c8f0-4953-b291-b4893959d5d9.mp4">|

For images:

| Before | After |
|--------|--------|
| ![before](https://github.com/LedgerHQ/ledger-live/assets/9203826/026c13c0-540c-486c-b5f7-b3858c153933) | ![after](https://github.com/LedgerHQ/ledger-live/assets/9203826/aa9666d6-f295-41ae-8d05-c6690c455423) | 

-->

```

Reply:
would add too much verbosity in the textarea i think, i rarely think we will need to do these before after images

Reply:
Since it's in comments I don't really see the drawbacks. On the contrary, if you need it, you uncomment it and / or copy paste, if you don't, you just keep it as comment.
But up to you in the end